webtransport/quic/handlers/client-indication.quic.py

```diff
@@ -0,0 +1,26 @@
+import asyncio
+import logging
+
+from aioquic.asyncio import QuicConnectionProtocol
+from aioquic.quic.events import QuicEvent
+from typing import Dict
+
+async def notify(connection):
+    _, writer = await connection.create_stream(is_unidirectional=True)
+    writer.write(b'PASS')
+    writer.write_eof()
```

Comment:
Is `connection` ever closed? Do we need to close it?

Reply:
It's not closed, and I don't think it needs to be closed in this case.